### PR TITLE
Made links more accessible in Search Suggestions

### LINKED ISSUE #650: More accessible links in the search suggestions
Instead of making the entire container a link which will be read out in full by screen readers, only use the heading. Using a clever overlay, clicking the whole container can still be preserved for visual users, like suggested in this video: https://youtu.be/qKRq-tFixWE?t=2227

This has also been discussed in #590.

### PR DESCRIPTION
**Pull request by Jaagrav**
Closes #650 
I have tested this with a screen reader extension by google, it works like expected

### CODE REVIEW

**Comment by Jaagrav** [comment]
Hey @zner0L still waiting for review...

**Comment by zner0L** [comment]
Hey @Jaagrav, are you still working on this?

**Comment by Jaagrav** [comment]
Hey @zner0L, sorry for the delay, we've got exams currently in our country, I know December is not a busy month for most around the world. I promise I'll get all my PRs merged by the end of this month, so please don't close them. 😅

I hope you understand.

**Comment by zner0L** [comment]
Yes, no worries. I was just wondering, bit I am glad to hear your still on it. Good luck for your exams!

File: src/Components/SearchBar.js [hunk 1]
```diff
@@ -102,19 +102,19 @@ if (Privacy.isAllowed(PRIVACY_ACTIONS.SEARCH)) {
                                           '"></span>'
                                         : '') +
                                     '</strong></span>' +
+                                    (this.props.anchorize ? '</a>' : '') +
                                     (d.runs?.length
-                                        ? '<br><span>' +
+                                        ? '<br><span aria-hidden="true">' +
```
[the remark this answers is not in the document]
Comment (Jaagrav) [reply src/Components/SearchBar.js:107]: image

Removing this aria-hidden, reads out the entire search result, so it reads the heading and the categories part as well. Is that how you'd like it? Then I will remove it.

[the remark this answers is not in the document]
Comment (zner0L) [reply src/Components/SearchBar.js:107]: Yes, exactly. I think users of screen readers can decide for themselves to skip that section if they want to.

**Review by zner0L — APPROVED** [review]
Thanks again! I hope you did well on your exams ;)

**Pull request merged**